monocle/worker.py

```diff
@@ -1264,9 +1269,33 @@ def normalize_gym(raw):
             'prestige': raw.gym_points,
```

Comment:
Will leave it for legacy reasons. Somebody should make a separate PR to remove prestige.